decoder.go

```diff
@@ -6,6 +6,20 @@ import (
 	"io"
 )
 
+const (
+	// MaximumRecordLength defines the maximum length in bytes, acceptable for records while decoding.
+	// This maximum prevents from excessive memory allocation for decoding.
+	// The value is derived from MAX_PKT_SIZ 65536 in sflow reference implementation
+	// https://github.com/sflow/sflowtool/blob/bd3df6e11bdf8261a42734c619abfe8b46e1202f/src/sflowtool.c#L4313
+	MaximumRecordLength = 65536
+
+	// MaximumHeaderLength defines the maximum length in bytes, acceptable for packet flow samples while decoding.
+	// This maximum prevents from excessive memory allocation for decoding.
+	// The value is derived from INM_MAX_HEADER_SIZE 256 in sflow reference implementation
+	// https://github.com/sflow/sflowtool/blob/bd3df6e11bdf8261a42734c619abfe8b46e1202f/src/sflowtool.h#L28
+	MaximumHeaderLength = 256
```

Comment:
I'm not sure about this one. I typically sample more than 256 bytes, and Brocade allows you to capture up to 1300 (see [Specifying the maximum flow sample size](http://www.brocade.com/content/html/en/administration-guide/fastiron-08030-adminguide/GUID-0841A65C-F58D-417F-B0E3-9846539ED13C.html)).

I think 1300 is a better choice.